src/components/domain/project/projectRegisterModal/step/Step3.tsx

```diff
@@ -58,20 +86,42 @@ export default function Step3() {
           </FormItem>
         )}
       />
-      <FormItem>
-        <FormLabel className="mobile1">기술스택</FormLabel>
-        <FormDescription className="text-gray-500 caption">
-          프로젝트에 사용된 기술스택을 입력해 주세요.
-        </FormDescription>
-        <div className="flex flex-wrap gap-1">
-          <TagInput
-            prefixChar="#"
-            className="tag-gray"
-            defaultValue="Spring Framework"
-            setValue={(value) => console.log(value)}
-          />
-        </div>
-      </FormItem>
+      <FormField
+        control={control}
+        name="skills"
+        render={({ field }) => (
+          <FormItem>
+            <FormLabel className="mobile1">기술스택</FormLabel>
+            <FormDescription className="text-gray-500 caption">
+              프로젝트에 사용된 기술스택을 입력해 주세요.
+            </FormDescription>
+            <FormControl>
+              <ul className="flex flex-wrap gap-1">
+                {field.value.map((skill, skillIndex) => (
+                  <li key={skillIndex}>
+                    <TagInput
+                      value={skill}
+                      colorTheme="gray"
+                      buttonType="delete"
+                      onClick={() => {
```

Comment:
여기서 선언하신 이유가 있나요? 어떤 함수인지 잘 모르겠어요. jsx 밖에서 선언해주는게 좋을 것 같습니다. 위에도 댓글 달아두었습니다. 

```suggestion
                      onClick={() => handleSkillDelete(field.value, skillIndex)}
```

Reply:
✅ fae9e7c86557468476bb00d8dd9f3ad607a336ac

이유는 없었습니다!